src/test/scala/edu/ie3/simona/test/common/input/NodeInputTestData.scala

```diff
@@ -96,4 +98,86 @@ trait NodeInputTestData extends DefaultTestData {
       GermanVoltageLevelUtils.HV,
       1,
     )
+
+  // PvModelITSpec node input models
+  val geometryFactory = new GeometryFactory()
+
+  val p1: Point = geometryFactory.createPoint(new Coordinate(10.3837, 53.9209))
```

Comment:
please directly include this below into the NodeInput. That way we don't need to assign a val for this.